api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Enabled through the `dom.webcomponents.enabled` preference in `about:config`"
+          },
+          "safari": {
+            "version_added": "10.1",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
```

Comment:
At first look, I think customized built-in elements should be a subfeature here.

Reply:
@teoli2003 are we certain? Because the implementors (vendors) have no immediate plan to support this (with the exception of Chrome). 

Please advise and I will update accordingly.

Reply:
@teoli2003 also would `.define()`, `.get()`, & `.whenDefined()` be considered "Sub-Features"?